### PR TITLE
Document new sections & move packages to new sections

### LINKED ISSUE #203: clean up utils section
the util and utils section are the largest sections and have some apps that can be re-sectioned. some ideas for sections:

* streaming
* runtimes (or libraries?)
* terminals
* input

### PR DESCRIPTION
**Pull request by raisjn**
add screensharing, templates, devel and admin sections

### CODE REVIEW

**Review by LinusCDE — DISMISSED** [review]
Seems good. Not sure what issue/pr was responsible for re-categorizing packages into these sections but they should probably get merged at around the same time.

**Comment by matteodelabre** [comment]
Do you think we could use this PR to actually move packages to their new sections @raisjn? So that we could close #203.

**Comment by raisjn** [comment]
do the package versions need to be bumped or is it enough to just change the section in the package file? (basically: do we need new packages or is it ok just to update the web listing)

**Comment by matteodelabre** [comment]
Would be best to update the revision number since this will change the `control` file inside the package, and therefore the package checksum.

**Comment by raisjn** [comment]
upon reflection, i don't have available time to bump versions and change packages - i'll leave it to someone else to merge this PR or to expand upon it

**Comment by matteodelabre** [comment]
Done. This will need a new review though.

**Review by LinusCDE — COMMENTED** [review]
Should we maybe quote all sections? I've seen that that spellcheck enforced that only on "admin", but having some quoted and some not might confuse people.

**Comment by matteodelabre** [comment]
Yeah, Shellcheck thought there was a possibility of a mistake since `admin` is also a command. I guess we can quote them all.

**Comment by matteodelabre** [comment]
Done.

**Review by LinusCDE — DISMISSED** [review]
Thanks. :shipit:

File: package/rmkit/package [hunk 1]
```diff
@@ -85,7 +85,7 @@ nao() {
     pkgdesc="Nao Package Manager: opkg UI built with SAS"
     url="https://rmkit.dev/apps/nao"
     pkgver=0.1.0-4
```
Comment (Eeems) [inline package/rmkit/package]:
```suggestion
    pkgver=0.1.0-5
```

File: package/rmkit/package [hunk 2]
```diff
@@ -131,7 +131,7 @@ simple() {
     pkgdesc="Simple app script for writing scripted applications"
     url="https://rmkit.dev/apps/sas"
     pkgver=0.1.1-1
```
Comment (Eeems) [inline package/rmkit/package]:
```suggestion
    pkgver=0.1.1-2
```

File: package/rm2-suspend-fix/package [hunk 3]
```diff
@@ -7,7 +7,7 @@ pkgdesc="Fix issue where suspend breaks networking on the reMarkable 2"
 url=https://toltec-dev.org/
 pkgver=0.0.0-1
```
Comment (LinusCDE) [inline package/rm2-suspend-fix/package]:
```suggestion
pkgver=0.0.0-2
```

**Review by LinusCDE — CHANGES_REQUESTED** [review]
Good catch, Eeems. I hadn't checked on that. Found another one as well.

File: package/rm2-suspend-fix/package (hunk 3 above)
Comment (Eeems) [reply package/rm2-suspend-fix/package]: I really thought I had found them all when I was skimming through this morning haha.

**Review by LinusCDE — APPROVED** [review]

**Review by Eeems — APPROVED** [review]
lgtm

**Comment by matteodelabre** [comment]
Thanks for your careful reviews. These things are hard to spot, and I’m still not sure what’s the best way to automatically detect whether a package has changes that require a version bump. Something we’ll need to figure out in #8.

**Pull request merged**